langlib/langlib-test/src/test/resources/test-src/arraylib_test.bal

```diff
@@ -707,7 +707,7 @@ function testShiftOnTupleWithoutValuesForRestParameter() {
     string detailMessage = message is error ? message.toString() : message.toString();
     assertValueEquality("{ballerina/lang.array}OperationNotSupported", err.message());
     assertValueEquality("the number of members in a tuple value should be greater " +
-    "than the member types of the tuple to perform a 'shift' operation", detailMessage);
+    "than the member types of the tuple to perform this operation", detailMessage);
```

Comment:
a 'shift' operation looks nice for me than saying this operation.

Reply:
It was changed because the same function and error was reused in more than one operation such as `shift` and `remove`. Will change the error to display the operation name by passing it as an argument to the function.

Reply:
Changes added in 70d6ff420ac8a0265d1361c26cc5b14b88a90b8c and commits proceeding that.